gamification-evm-webapp/src/main/webapp/vue-app/connectorEventExtensions/components/EvmEventForm.vue

```diff
@@ -177,7 +190,17 @@ export default {
     },
     emptyERC20Token() {
       return !this.typing && this.contractAddress && !this.erc20Token;
-    }
+    },
+    explorerLink() {
+      const url = this.selected?.providerUrl.substring(this.selected.providerUrl.indexOf('//') + 2, this.selected.providerUrl.indexOf('.g.alchemy.com'));
```

Comment:
what if we use a different EVM access provider ?

Reply:
You are right!
fixed

Reply:
can we compute and get the `networkId` from `this.properties` that should be computed and stored in Backend after adding the Network URL ? in fact, this will be helpful to be completely independent from Blockchain Access URL.

Reply:
Done!